### DISCUSSION TITLE
Delete post

### DISCUSSION THREAD

**Discussion by ghost** [skyline-emu/skyline#381] [General]
Please use open gl 2.5
Vulkan is a form of planned obsolescence and a bussiness practice to push sales of newer devices
Not everyone has money or resources to buy vulkan supported device
Theres no way to know vulkan support before purchase
Please support more users.Dont be greedy with your business

**Comment by bylaws** [comment]
Idk who you are but if this is one heck of a gem, thanks for making my day
